PlutoHelperAgent/AppDelegate.m

```diff
@@ -20,6 +20,8 @@ @implementation AppDelegate
 @synthesize statusBar;
 @synthesize connectionWorking;
 
+int connectionAttempts = 0;
```

Comment:
really not nice, but i'll take it for the moment.
this should be in the object state, not as an application global.